.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -33,6 +33,9 @@ jobs:
         run: |
           bash ./scripts/ci-install-${{ matrix.os }}.sh
 
+      - name: Install tree-sitter-cli
+        run: npm install -g tree-sitter-cli
```

Comment:
We can also pin this to the current version, but maybe is good to help testing new releases?

Reply:
better stay up-to-date